src/serlo-editor/plugins/text/types/text-editor.ts

```diff
@@ -19,7 +19,7 @@ export interface Heading {
 
 export interface Paragraph {
   type: 'p'
-  children: (CustomText | MathElement)[] // TODO Restrict this to only inline MathElement
+  children: (CustomText | MathElement)[] // TODO: Restrict this to only inline MathElement
```

Comment:
@LarsTheGlidingSquirrel  should we keep this TODO or turn it into an issue / normal comment?